ivy/functional/frontends/tensorflow/nest.py

```diff
@@ -1,3 +1,4 @@
+# for review
```

Comment:
replace repeated `if` calls with `if` `elif`

Reply:
could you also remove the `if isinstance(structure, tf.RaggedTensor):` section inside the `flatten` implementation? Can tensorflow not also be handled correctly inside `_flatten_sparse_array`?